<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">QFrame{
background-color: rgb(247, 247, 233) ;
color: rgb(176, 131, 63);
}
QPushButton{
background-color:#000000ff;
border-radius:25px;
}
QPushButton:hover{
background-color:rgb(230, 225, 210);
border-radius:25px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="1,6,1">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frameTitulo">
         <property name="styleSheet">
          <string notr="true">QFrame{
background-color: rgb(247, 247, 233) ;
color: rgb(176, 131, 63);
}
QPushButton{
background-color:#000000ff;
border-radius:25px;
}
QPushButton:hover{
background-color:rgb(230, 225, 210);
border-radius:25px;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QPushButton" name="botonSalir">
          <property name="geometry">
           <rect>
            <x>740</x>
            <y>0</y>
            <width>51</width>
            <height>51</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>icono-salir.png</normaloff>icono-salir.png</iconset>
          </property>
         </widget>
         <widget class="QPushButton" name="botonRegresar">
          <property name="geometry">
           <rect>
            <x>690</x>
            <y>0</y>
            <width>51</width>
            <height>51</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>icono-atras.png</normaloff>icono-atras.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
         </widget>
         <widget class="QLabel" name="Titulo">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>281</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>22</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Nueva Receta</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame{ background-color: rgb(247, 247, 233) ; }
QRadioButton {
    color: rgb(200, 160, 100);      
}

QRadioButton::indicator {
    width: 14px;
    height: 14px;
    border-radius: 7px;
    border: 2px solid rgb(230, 150, 150);
    background-color: rgb(255, 255, 255);
}

QRadioButton::indicator:checked {
    border: 2px solid rgb(220, 100, 100);
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="10,1">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="frameReceta">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="labelCategoria">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Categoria</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="categorias"/>
             </item>
             <item>
              <widget class="QLabel" name="labelNombre">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Nombre Receta</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="nombreReceta"/>
             </item>
             <item>
              <widget class="QLabel" name="labelUnidadesMetricas">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Unidades Metricas</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="unidadesMetricas"/>
             </item>
             <item>
              <widget class="QLabel" name="labelIngredientes">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Ingredientes:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTextEdit" name="listaIngredientes"/>
             </item>
             <item>
              <widget class="QLabel" name="labelProcedimiento">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Procedimiento:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTextEdit" name="procedimiento"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frameBotones">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QPushButton" name="botonGuardar">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>10</y>
               <width>45</width>
               <height>45</height>
              </rect>
             </property>
             <property name="minimumSize">
              <size>
               <width>45</width>
               <height>45</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>icono-guardar.png</normaloff>icono-guardar.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
            </widget>
            <widget class="QPushButton" name="botonEliminar">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>70</y>
               <width>40</width>
               <height>40</height>
              </rect>
             </property>
             <property name="minimumSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>icono-eliminar.png</normaloff>icono-eliminar.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>35</width>
               <height>35</height>
              </size>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frameInfo">
         <property name="styleSheet">
          <string notr="true">QFrame {
    background-color: rgb(247, 247, 233);
}
QPushButton {
    background-color: rgb(247, 247, 233);  
    color: rgb(200, 160, 100);                 
    border-radius: 20px;
    padding: 8px 16px;
    font-weight: bold;
    border: 1px solid rgb(210, 200, 180);  
}

QPushButton:hover {
    background-color: rgb(240, 235, 220);  
	color: rgb(230, 225, 210);
}

QPushButton:pressed {
    background-color: rgb(220, 215, 200);  
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QPushButton" name="botonInfo">
          <property name="geometry">
           <rect>
            <x>740</x>
            <y>0</y>
            <width>51</width>
            <height>51</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QFrame{
background-color: rgb(247, 247, 233) ;
}
QPushButton{
background-color:#000000ff;
border-radius:25px;
}
QPushButton:hover{
background-color:rgb(230, 225, 210);
border-radius:25px;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>icono-informacion.png</normaloff>icono-informacion.png</iconset>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
